stream-chat-android-client/src/main/java/io/getstream/chat/android/client/clientstate/ClientStateService.kt

```diff
@@ -52,13 +52,7 @@ internal class ClientStateService {
             state<ClientState.User.Pending.WithToken> {
                 onEvent<ClientStateEvent.ConnectedEvent> { state, event -> ClientState.User.Authorized.Connected(event.connectionId, event.user, state.token) }
                 onEvent<ClientStateEvent.DisconnectedEvent> { _, _ -> stay() }
-                onEvent<ClientStateEvent.ConnectedEvent> { state, event ->
-                    ClientState.User.Authorized.Connected(
-                        event.connectionId,
-                        event.user,
-                        state.token
-                    )
-                }
```

Comment:
I get the new line being added here, but why is the handling for `ConnectedEvent` being removed?

Reply:
Maybe I don't get something about state machines, but the same event is handled above:
```
onEvent<ClientStateEvent.ConnectedEvent> { state, event -> 
      ClientState.User.Authorized.Connected(event.connectionId, event.user, state.token) }
```

Reply:
By some reason it seems taht was duplicated

Reply:
Yeah, it was dublicated. Anyway there is a test for such case. It helps us not to break it
(`Given user pending with token state When connected Should move to user connected state`)